client/components/SurrenderForm.js

```diff
@@ -0,0 +1,197 @@
+import { set } from 'lodash';
+import React, { useState } from 'react'
+
+const SurrenderForm = () => {
+  const [surrenderData, setSurrenderData] = useState({
+    name: "",
+    phoneNumber: "",
+    email: "",
+    petName: "",
+    petAge: "",
+    petType: "Select Pet Type",
+    petImage: "",
+    vaccinationStatus: "",
+  })
+  const [posted, setPosted] = useState(false)
+  const [errors, setErrors] = useState([])
+
+  let errorContainer = null
+  if (errors.length !== 0) {
+    errorContainer = errors.map(error => {
+      switch (error) {
+        case "phoneNumber":
+          error = "phone number";
+          break;
+        case "petType":
+          error = "pet type";
+          break;
+        case "petAge":
+          error = "pet age";
+          break;
+        case "petImage":
+          error = "pet image";
+          break;
+        case "petName":
+          error = "pet name";
+          break;
+        case "vaccinationStatus":
+          error = "vaccination status";
+          break;
+      }
+      return <p>{`${_.capitalize(error)} is empty or not selected.`}</p>
+    })
+  }
+
+  const checkForErrors = () => {
+    const errorsArray = []
+    const fieldsData = Object.keys(surrenderData)
```

Comment:
Take a look at the `React Form Validations with Hooks` article for how we can use lodash to help generate errors